.github/workflows/presubmit.yml

```diff
@@ -25,13 +16,9 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.actor }}-${{ github.ref_name }}
   cancel-in-progress: true
 
-permissions:
-  id-token: write
-  contents: read
-
 jobs:
-  format-lint-test:
-    name: Checks format, lints, runs tests, and checks coverage
+  format-lint-and-test:
```

Comment:
Not that it really matters, but are we linting PRs twice then? Once on submission, once again when pushing to develop or main?

Reply:
That's right. The risk I'm trying to mitigate with that is folks pushing directly to the develop and/or main branches. Until we can get branch protection rules in place this remains a possibility. I know this because I screwed up and pushed directly to the develop branch /facepalm